bzt/modules/blazemeter.py

```diff
@@ -1229,7 +1229,13 @@ def start_if_ready(self):
     def stop_test(self):
         if self.master:
             self.log.info("Ending cloud test...")
-            self.master.stop()
+            if not self._last_status:
+                self.get_master_status()
+
+            if self._last_status["progress"] >= 100:
```

Comment:
Not really important in this PR, but does BlazeMeter describe "progress" values anywhere? It might be nice to have a set of constants like `PROGRESS_DOWNLOADING_IMAGE`, `PROGRESS_BOOTING`, `PROGRESS_RUNNING_TEST`, etc in our BZA client.